users/app.py

```diff
@@ -54,15 +54,47 @@ def add_update_user():
     return jsonify(user_object), (201 if added_new_user else 200)
 
 
+def upsert_user_in_db(user_object, users_collection):
+    """Updates or inserts the user object into user_collection.
+
+    Adds the is_oganizer field to user_object with default value False.
```

Comment:
`is_organizer` -- any reason to call out this field in particular? Do you want to simply say "Sets defaults if needed"?

Reply:
Done https://github.com/knative-portability/large-events/pull/8/commits/aa11d3de295c56d34f9d1a2853dc97c822ec2032